src/base/utils/foreignapps.cpp

```diff
@@ -262,32 +265,34 @@ PythonInfo Utils::ForeignApps::pythonInfo()
 
         if (!pyInfo.isValid())
         {
-            const QString exeNames[] = {u"python3"_s, u"python"_s};
-            for (const QString &exeName : exeNames)
+            const Path exeNames[] = {Path(u"python3"_s), Path(u"python"_s)};
```

Comment:
I believe we really talk about "names" here. I would revert this change and convert "name" to `Path` where it is needed, i.e. `if (testPythonInstallation(Path(exeName), pyInfo))`.

Reply:
>I believe we really talk about "names" here. I would revert this change and convert "name" to Path where it is needed, i.e. if (testPythonInstallation(Path(exeName), pyInfo)).

All done.
To be honest I don't really see this would be different. It will be converted to Path eventually, and perhaps in other cases it will loose the opportunity to construct at compile time.